src/index.js

```diff
@@ -81,17 +129,32 @@
       }
       return el;
     },
+    getElByDataAttribute: (attribute, value, scope = document) => {
+      const el = scope.querySelector(`[data-${attribute}="${value}"]`)
```

Comment:
you can use early return here
```suggestion
      return scope.querySelector(`[data-${attribute}="${value}"]`)
```